File: php/test/ExistsTest.php
<?php
declare(strict_types=1);

// Bng2latlongConverter SDK exists test

require_once __DIR__ . '/../bng2latlongconverter_sdk.php';

use PHPUnit\Framework\TestCase;

class ExistsTest extends TestCase
{
    public function test_create_test_sdk(): void
    {
        $testsdk = Bng2latlongConverterSDK::test(null, null);
        $this->assertNotNull($testsdk);
    }
}

File: rb/test/exists_test.rb
# Bng2latlongConverter SDK exists test

require "minitest/autorun"
require_relative "../Bng2latlongConverter_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = Bng2latlongConverterSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
